clang/lib/StaticAnalyzer/Checkers/CStringChecker.cpp

```diff
@@ -150,6 +150,8 @@ class CStringChecker : public Checker< eval::Call,
       // FIXME: C23 introduces 'memset_explicit', maybe also model that
       {{CDM::CLibraryMaybeHardened, {"strcpy"}, 2},
        &CStringChecker::evalStrcpy},
+      {{CDM::CLibraryMaybeHardened, {"lstrcpy"}, 2},
```

Comment:
```suggestion
      {{CDM::CLibrary, {"lstrcpy"}, 2},
```
The matching mode `CLibraryMaybeHardened` was introduced to also recognize references to the "hardened" function variants that are named like `_strcpy_chk` or `__builtin___strcpy_chk` (and may have one or two additional parameters compared to the "regular" function). As `lstrcpy` does not have a hardened variant (according to a google search that produced zero results), we should simply match it with `CDM::CLibrary`.
In this case the plain `CLibrary` matching mode is enough, because